.github/workflows/go.yml

```diff
@@ -0,0 +1,46 @@
+name: Push
+
+on:
+  push:
+    branches: [ master ]
+    tags:
+      - '*'
+
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.14
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.14
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Get dependencies
+      run: |
+        go get -v -t -d ./...
+        if [ -f Gopkg.toml ]; then
```

Comment:
looks like we don't need L28-31 since we're not using `dep`. Actually, do we even need this "Get dependencies" step? Modules should be pulled automatically during the build, right?

Reply:
Yes, it shouldn't be necessary. the GH template gave it to me and I didn't remove it.
Will remove it.